### PR TITLE
Element Styling - items and new item form

### PR DESCRIPTION
**Pull request by Joseph-Cashmore**
I have styled the item elements and the "add new item" form. Currently, we still have the delete button appear with the item by default, but more functionality needs to be developed before this can really change.

### CODE REVIEW

File: src/components/ShoppingList/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    background-color: var(--item_grey);
+    color: var(--font_grey);
+    height: 40px;
+    border-radius: 20px;
+    margin-top: 10px;
+    box-sizing: border-box;
+    padding: 0 10px 0 28px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```
Comment (Joseph-Cashmore) [inline src/components/ShoppingList/styles.module.css:15]: The new item form and the item use a lot of the same css attributes, but are currently sourced from different files. Should this kind of thing be global? Or is it fine how it is?

Comment (samstradling) [reply src/components/ShoppingList/styles.module.css:15]: I think this is a sign that the wrapper at least should be the same component, this looks good for now though and we can refactor it later to re-use the same code more

File: src/components/ShoppingList/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+.newItem button[type=submit] {
+    background-color: transparent;
+    outline: none;
+    border: none;
+}
+
+.newItem input {
+    border-bottom: 2px solid var(--main_green);
+    font-size: 12px;
+    font-weight: 700;
```
Comment (samstradling) [inline src/components/ShoppingList/styles.module.css:28]: Does this need a `flex-grow: 1;` so it doesn't start scrolling for longer items?
long names mov

File: src/style/index.css [hunk 3]
```diff
@@ -12,6 +12,12 @@ code {
     monospace;
 }
 
+:root {
+  --item_grey: #F8F8F8;
+  --font_grey: #5B5B5B;
+  --main_green: #8AB661;
+}
```
Comment (samstradling) [inline src/style/index.css:19]: Nice 👍

**Review by samstradling — APPROVED** [review]
Looks great!

File: src/components/ShoppingList/styles.module.css (hunk 2 above)
Comment (Joseph-Cashmore) [reply src/components/ShoppingList/styles.module.css:28]: Most likely! I never actually typed in a longer name to test it 🤦‍♂️

**Pull request merged**